actions/transfer-case.vbs

```diff
@@ -359,45 +358,30 @@ ELSE 'this means out of county is TRUE '
             call create_MAXIS_friendly_date(excluded_time_begin_date, 0, 6, 28)
             EMWriteScreen hc_cfr, 15, 39
         END IF
-        IF excluded_time_begin_date = "" AND excluded_time_dropdown = "NO" THEN
-            EMWriteScreen "__", 6, 28
-            EMWriteScreen "__", 6, 31
-            EMWriteScreen "__", 6, 34
+        IF excluded_time_dropdown = "NO" THEN
+            CALL clear_line_of_text(6, 28) 'clearing any old Excluded Time Begin Date __ __ __'
+            CALL clear_line_of_text(6, 31)
+            CALL clear_line_of_text(6, 34)
+            CALL clear_line_of_text(11, 39) 'Clearing Current Fin Resp County Cash I'
+            CALL clear_line_of_text(12, 39) 'Cash II'
+            CALL clear_line_of_text(13, 39) 'GRH'
+            CALL clear_line_of_text(14, 39) 'Health Care'
+            CALL clear_line_of_text(15, 39) 'MA Excluded Time'
+            CALL clear_line_of_text(16, 39) 'IV-E Foster Care
         END IF
-        IF manual_cfr_hc_checkbox = CHECKED THEN
-            EMWriteScreen hc_cfr, 14, 39
-            EMWriteScreen hc_cfr_month, 14, 53
-            EMWriteScreen hc_cfr_year, 14, 59
-        ELSE
-            IF ma_case = True THEN
-                EMWriteScreen hc_cfr, 14, 39
-                EMWriteScreen cfr_month, 14, 53
-                EMWriteScreen cfr_year, 14, 59
-            END IF
+        IF (ga_case = TRUE or msa_case = TRUE or mfip_case = TRUE or dwp_case = TRUE or grh_case = TRUE) THEN 'previously we read PROG for cash one and cash two programs unsure if this is necessary'
+            EMWriteScreen cash_cfr, 11, 39
+            EMWriteScreen cfr_month, 11, 53
```

Comment:
The month and year variables don't match the dialog variables here.

Reply:
got it, i had to keep the hc_cfr_month variable incase it is different from the cash per the POLI